interface/orders/types_edit.php

```diff
@@ -72,81 +71,147 @@ function recursiveDelete($typeid)
-        function del_related(s) {
-            my_del_related(s, document.forms[0][rcvarname], false);
-        }
+.ordonly {
+    
+}
 
-        // This invokes the find-code popup.
-        function sel_related(varname) {
-            if (typeof varname == 'undefined') varname = 'form_related_code';
-            rcvarname = varname;
-            dlgopen('../patient_file/encounter/find_code_dynamic.php', '_blank', 900, 600, false, 'Find Code');
-        }
+.resonly {
+    
+}
+.label-div > a {
+    display:none;
+}
+.label-div:hover > a {
+   display:inline-block; 
+}
+div[id$="_info"] {
+    background: #F7FAB3;
+    padding: 20px;
+    margin: 10px 15px 0px 15px;
+}
+div[id$="_info"] > a {
+    margin-left:10px;
+}
+@media only screen {
+        fieldset > [class*="col-"] {
+        width: 100%;
+        text-align:left!Important;
+    }
+}
+/*@media only screen and (max-width: 450px) {
+        #button-container > [class*="col-"] {
+        width: 100%;
+        text-align:left!Important;
+    }*/
+</style>
+
+<script language="JavaScript">
+
+<?php require($GLOBALS['srcdir'] . "/restoreSession.php"); ?>
+
+// The name of the form field for find-code popup results.
+var rcvarname;
+
+// This is for callback by the find-code popup.
+// Appends to or erases the current list of related codes.
+function set_related(codetype, code, selector, codedesc) {
+    var f = document.forms[0];
+    var s = f[rcvarname].value;
+    if (code) {
+        if (s.length > 0) s += ';';
+        s += codetype + ':' + code;
+    } else {
+        s = '';
+    }
+    f[rcvarname].value = s;
+}
 
-        // Show or hide sections depending on procedure type.
-        function proc_type_changed() {
-            var f = document.forms[0];
-            var pt = f.form_procedure_type;
-            var ix = pt.selectedIndex;
-            if (ix < 0) ix = 0;
-            var ptval = pt.options[ix].value;
-            var ptpfx = ptval.substring(0, 3);
-            $('.ordonly').hide();
-            $('.resonly').hide();
-            if (ptpfx == 'ord') $('.ordonly').show();
-            if (ptpfx == 'res' || ptpfx == 'rec') $('.resonly').show();
-        }
+// This is for callback by the find-code popup.
+// Returns the array of currently selected codes with each element in codetype:code format.
+function get_related() {
+ return document.forms[0][rcvarname].value.split(';');
+}
 
-        $(document).ready(function () {
-            proc_type_changed();
-        });
+// This is for callback by the find-code popup.
+// Deletes the specified codetype:code from the currently selected list.
+function del_related(s) {
+ my_del_related(s, document.forms[0][rcvarname], false);
+}
 
-    </script>
+// This invokes the find-code popup.
+function sel_related(varname) {
+ if (typeof varname == 'undefined') varname = 'form_related_code';
+ rcvarname = varname;
+ dlgopen('../patient_file/encounter/find_code_dynamic.php', '_blank', 900, 600);
```

Comment:
This reverted something @sjpadgett recently did

Reply:
Not as simple as I thought, needs translation as well ;)